PodcastMenu/OvercastController.swift

```diff
@@ -41,12 +41,41 @@ class OvercastController: NSObject, WKNavigationDelegate {
         return WKUserScript(source: source, injectionTime: .atDocumentEnd, forMainFrameOnly: false)
     }()
     
+     fileprivate func startAutomaticRefresh() {
+        
+        Timer.scheduledTimer(timeInterval: TimeInterval(Constants.automaticRefreshInterval), target: self, selector: #selector(self.refresh(timer:)) , userInfo: nil, repeats: true)
+        
+        NSWorkspace.shared().notificationCenter.addObserver(forName: Notification.Name.NSWorkspaceDidWake, object: NSWorkspace.shared(), queue: nil) { [weak self] _ in
+            
+            self?.refreshPodcastsIfNeeded()
+        }
+    }
+    
+    @objc fileprivate func refresh(timer: Timer) {
+        
+        refreshPodcastsIfNeeded()
+    }
+        
+    fileprivate func refreshPodcastsIfNeeded() {
+        
+       guard (activity == nil) else { return }
+       guard (self.webView.url != nil) else { return }
+       guard self.webView.url?.path == Constants.homePath else { return }
+       guard !self.webView.isLoading else { return }
+       guard !(self.webView.window?.isVisible)! else { return }
```

Comment:
Please remove the force unwrap. You can use `guard self.webView.window?.isVisible == false` ;)